giru/adapters_fs.py

```diff
@@ -0,0 +1,88 @@
+import csv
```

Comment:
These implementation were just taken as is from the old `repliers.py` and `commands.py`